### DISCUSSION TITLE
Problem with default wallet list when connecting

### DISCUSSION THREAD

**Discussion by NichaelScofield** [WalletConnect/walletconnect-monorepo#3464] [Developer Support] [answered]
The version of walletconnect v1 is to detect which wallets are installed and recommend it first, but walletconnect v2 cancels this. Please see how to do it, which is very confusing for your own users. Or tell us how to let our users see our wallet by default
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/142303803/60a10937-9815-44ac-a052-d695b66f8fc5)

**Comment by glitch-txs** [comment, marked as answer]
Check the options section in our documentation :)

https://docs.walletconnect.com/2.0/
